src/rendervulkan.cpp

```diff
@@ -2661,6 +2660,22 @@ bool vulkan_composite( struct Composite_t *pComposite, struct VulkanPipeline_t *
 		vkCmdPipelineBarrier( curCommandBuffer, VK_PIPELINE_STAGE_COMPUTE_SHADER_BIT, VK_PIPELINE_STAGE_COMPUTE_SHADER_BIT,
 			      		0, 0, nullptr, 0, nullptr, 1, &memoryBarrier );
 
+		// Budge other layers down if we included the override redirect to blur before.
+		if (blur_layer_count == 1)
+		{
+			for (int i = 2; i < blurComposite.nLayerCount; i++)
+			{
+				blurComposite.data.vScale[i - 1] = blurComposite.data.vScale[i];
+				blurComposite.data.vOffset[i - 1] = blurComposite.data.vOffset[i];
+				blurComposite.data.flOpacity[i - 1] = blurComposite.data.flOpacity[i];
+
+				blurLayers.layerBindings[i - 1] = std::move(blurLayers.layerBindings[i]);
+			}
+
+			blurComposite.data.nBorderMask = (blurComposite.data.nBorderMask & 0b1) | ((blurComposite.data.nBorderMask & 0b11111100) >> 1);
+			blurComposite.nLayerCount--;
+		}
+
```

Comment:
Doesn't this break the conditional blur when the override layer is the top level layer at a location? Because in the situation we won't use the image created by the horizontal pass.

Reply:
Afaict the result would be that the override layer is completely missing then.

Reply:
Uhhh pershaps, will check.

Reply:
Should be fixed now!